src/lando/static_src/legacy/js/components/Timeline.js

```diff
@@ -1,50 +1,38 @@
-'use strict';
+'e strict';
```

Comment:
Was this clipped by accident?

Reply:
Looks like it... And it still works. Oh Javascript...